libyul/AsmAnalysis.h

```diff
@@ -104,20 +104,14 @@ class AsmAnalyzer
 private:
 	/// Visits the expression, expects that it evaluates to exactly one value and
 	/// returns the type. Reports errors on errors and returns the default type.
```

Comment:
The comment is out of date.
```suggestion
	/// Visits the expression, expects that it evaluates to exactly one value.
	/// Reports errors otherwise.
```